crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -178,7 +178,8 @@ pub struct Node<S: TenTenOneStorage, N: Storage> {
     pub dlc_storage: Arc<DlcStorageProvider<S>>,
 
     // fields below are needed only to start the node
-    listen_address: SocketAddr,
+    #[allow(dead_code)]
+    listen_address: SocketAddr, // Irrelevant when using websockets
```

Comment:
❓Can we feature-flag this instead or is it too much of a pain?

Reply:
We could, but I thought that perhaps we may want to use it somewhere else (?). If not I can definitely get rid of it

Reply:
Not a big deal, but I think this can be safely removed (from the version that doesn't use it).